schedule/yast/yast2_ncurses/yast2_ncurses_opensuse.yaml

```diff
@@ -0,0 +1,33 @@
+---
+name:           yast2_ncurses
+description:    >
+  Test for yast2 UI, ncurses only. Running on created gnome
+  images which provides both text console for ncurses UI tests
+  as well as the gnome environment for the GUI tests.
+vars:
+  YUI_REST_API: 1
+schedule:
+  - boot/boot_to_desktop
+  - console/system_prepare
+  - console/prepare_test_data
+  - console/consoletest_setup
+  - console/setup_libyui_running_system
+  - console/yast2_rmt
+  - console/yast2_ntpclient
+  - console/yast2_tftp
+  - console/yast2_proxy
+  - console/yast2_vnc
+  - console/yast2_http
+  - console/yast2_ftp
+  - console/yast2_apparmor
+  - console/yast2_lan
+  - console/yast2_lan_device_settings
+  - console/yast2_lan_hostname_dhcp_no
+  - console/yast2_lan_hostname_dhcp_yes
+  - console/yast2_lan_hostname_dhcp_yes_eth
+  - console/yast2_dns_server
+  - console/yast2_nfs_client
+  - console/yast2_snapper_ncurses
+test_data:
+  ui: ncurses
```

Comment:
test data should be nested in some node related with what is suppose to drive. So when more data is added is less messy.